.github/workflows/compile_time_options.yml

```diff
@@ -36,6 +36,7 @@ jobs:
             $([ -z $GITHUB_TOKEN ] || echo "--config=remote-ci-linux") \
             --define=admin_html=enabled \
             --define=envoy_mobile_request_compression=disabled \
+            --define=envoy_enable_http_datagram=disabled \
```

Comment:
It's important to verify we have a build with H3 enabled (false here) and datagrams disabled.  Mind adding a comment here that the mobile build verifies that this works?